apps/dashboard/src/utils/constants.ts

```diff
@@ -28,6 +28,14 @@ export const STEP_TYPE_LABELS: Record<StepTypeEnum, string> = {
   [StepTypeEnum.CUSTOM]: 'Custom',
 };
 
+export const AUTO_OPEN_DRAWER_AFTER_CREATION_STEP_TYPES: StepTypeEnum[] = [
```

Comment:
Every TEMPLATE_CONFIGURABLE_STEP_TYPES should allow for the same behavior. So we don't need a separate enum for this.

Moreover, these enums have a bit of a code smell as the UI around drawers should work independently of the step types. We will get back to that soon; meanwhile, let's not add another constant array of array steps.